site/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "devDependencies": {
+    "@symfony/webpack-encore": "^0.27.0"
+  },
+  "dependencies": {
+    "node-sass": "^4.12.0",
+    "sass-loader": "^7.1.0"
+  }
+}
```

Comment:
Should there be a package lock file as well?

Reply:
Why do we need a `package lock`. We have a `yarn.lock`.